scripts/judges_server.py

```diff
@@ -0,0 +1,89 @@
+import rospy
+from ros_interop.srv import Team,TeamResponse,Odlc,OdlcResponse
+from ros_interop.msg import *
+
+
+class Teams():
```

Comment:
Each of these classes should be its own file under a `src` folder, then the script would just construct these objects and spin.